.github/workflows/wolfi-presubmit.yaml

```diff
@@ -68,7 +68,10 @@ jobs:
           - guac
           - mdbook
           - s3cmd
-          - db # Uses license-path
+          - db  # Uses license-path
+          - cadvisor  # uses cgroups
+          - dotnet-9  # huge workspace size 3+gb
+          - fping  # uses get/setcaps
```

Comment:
Looks like this is already below 🤔 

I think what this is missing is running tests!

Reply:
Added the test case, now we need to merge this: https://github.com/wolfi-dev/os/pull/52948
to fix the xattr losing